### ISSUE TITLE
training the net again given output ranging between 0.4 and 0.5 only

### ISSUE THREAD

**Issue by aakash-saboo** [jfhealthcare/Chexpert#13]
When I am training the model again, it is giving probability values for consolidation between 0.4 and 0.5. and for other labels, the difference between max and min probability is again 0.1.
The AUC of consolidation is coming to be 0.945
Can you comment on this please?

Thank you

**Comment by deadpoppy** [comment]
the AUC just computes the rank of your predictions. independent of probability. @aakash-saboo

**Comment by aakash-saboo** [comment]
I do realise that it computes the rank of the prediction. But training with the exact same parameters and getting such a small range of values is very odd. The given pretrain weights doesnt give probability in such a narrow range.
Can you somehow tell me that with the exact same parameters as in example.json, is this supposed to happen?

**Comment by deadpoppy** [comment]
Sorry, I have no such problem. And I'd love to know why you have this problem. Is that an interesting finding.  @aakash-saboo

**Comment by aakash-saboo** [comment]
Hey.
By any chance do you think its because of batch normalization on multi GPUs? 
I am training on DGX1.
I just started training on 1 GPU only and it is now giving me values from 0.3 to 0.9.
Will share final results soon
Would love your comment on such a thing.

**Comment by aakash-saboo** [comment]
Hi. 
So this problem is only coming when using 'FPA' attention. I tried 'SAM' and it is giving output between 0 and 1.
